pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -105,5 +105,7 @@ buildPythonPackage rec {
       "x86_64-darwin"
     ];
     maintainers = [ maintainers.sheepforce ];
+    # Tests fail
+    broken = true;
```

Comment:
To unbreak PySCF, add to `pytestFlagsArray`:
  * `"--ignore-glob=pyscf/nac/test/test_sacasscf.py"`
   * `"--ignore-glob=pyscf/grad/test/test_casscf.py"`

and add to `disabledTests`

  * `"test_sacasscf_grad"`

These tests fail due to deprecated API calls, not because they are wrong in the numerical sense.